### DISCUSSION TITLE
Programing Language Choices.

### DISCUSSION THREAD

**Discussion by KragonTsal** [amidaware/tacticalrmm#637] [General]
Why did you choose the languages for this project? For the agent side why not use something like C#?

**Comment by wh1te909** [comment]
golang compiles to single binary with 0 dependencies. agent was originally written in python but rewritten in go because python is pain to ship esp on windows and most agent related issues were due to deployment while we were still on python. golang has solved this.

also with linux and mac agents in the works i don't think c# would be the best choice for these 2 OS
